examples/crypto-usage-tiingo-pandas.ipynb

```diff
@@ -0,0 +1,131 @@
+#%%
+#%% md
+
+# Tiingo Python for Cryptocurrencies
+
+
+This notebook shows basic usage of the `tiingo-python` library. If you're running this on `mybinder.org`, you can run this code without installing anything on your computer. You can find more information about what available at the [Tiingo website](https://api.tiingo.com/docs/general/overview), but this notebook will let you play around with real code samples in your browser.
+
+If you've never used `jupyter` before, I recommend this [tutorial](https://www.datacamp.com/community/tutorials/tutorial-jupyter-notebook) from Datacamp.
+
+#%% md
+
+## Basic Setup
+
+First, you'll need to provide your API key as a string in the cell below. If you forget to do this, the notebook cannot run. You can find your API key by visiting [this link](https://www.tiingo.com/account/api/token) and logging in to your Tiingo account.
+
+#%%
+
+!pip3 install os
```

Comment:
All the dependencies for the demo notebooks can be included here rather than in inline cells:

https://github.com/hydrosquall/tiingo-python/blob/master/binder/requirements.txt